conf/config.toml

```diff
@@ -142,6 +142,13 @@
 ## PD avoids migrating data to this store as much as possible.
 # low-space-ratio = 0.8
 
+## configure a specific store limit, the store id need to be an integer, here using 100
```

Comment:
These configs can only take effect on starting PD for the first time. So we strongly don't recommend setting up it since it's hard to know the store ID at the very beginning. But it's useful for gitops.